chronos/__init__.py

```diff
@@ -26,13 +26,16 @@
 import socket
 import json
 import logging
+import warnings
 
 # Python 3 changed the submodule for quote
 try:
     from urllib import quote
 except ImportError:
     from urllib.parse import quote
 
+SCHEDULER_VERSIONS = ('v1',)
```

Comment:
@Rob-Johnson I thought about this, but since chronos appears to have multiple APIs and they are only versioning ``/scheduler/*`` (they don't prefix ``/metrics``, for example) that's why I decided to use this variable. Happy to change it to whatever name makes the most sense, naming is hard 😊